codecoverage.py

```diff
@@ -238,7 +236,10 @@ def main():
             download_grcov()
             grcov_path = './grcov'
 
-        generate_report(grcov_path, 'lcov', 'output.info')
+        if args.with_artifacts:
```

Comment:
You can do:
```
generate_report(grcov_path, 'lcov', 'output.info', args.with_artifacts)
```

This way if args.with_artifact is None, generate_report should use the default 'ccov-artifacts'

Reply:
Right, forget my suggestion.